observatory-platform/observatory/platform/cli/generate_command.py

```diff
@@ -221,6 +255,38 @@ def generate_workflow(self, project_path: str, package_name: str, workflow_type:
                 f.write(f'    {workflow_module} = "{workflow_module}"\n')
             print(f"- Updated the identifiers file: {identifiers_dst_file}")
 
+    def generate_local_config_interactive(self, config_path: str, *, workflows: Set[str], oapi: bool):
```

Comment:
Should workflows be a 'List[str]' instead? From the usage/tests it seems that it a list is passed on.